### DISCUSSION TITLE
Error: Could not parse CSS stylesheet

### DISCUSSION THREAD

**Discussion by iamyegor** [primefaces/community#410] [PrimeReact] [answered]
[This is super simple project that can provide you with more context](https://github.com/iamyegor/primeReact-Error-Could-not-parse-CSS-stylesheet-while-testing)

I'm using vitest + react testing library for testing. When I test a component which contains prime react component, test successfully pases, but in console I get the following error:

`Error: Could not parse CSS stylesheet
    at exports.createStylesheet (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/helpers/stylesheets.js:37:21)
    at HTMLStyleElementImpl._updateAStyleBlock (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/nodes/HTMLStyleElement-impl.js:68:5)
    at HTMLStyleElementImpl._childTextContentChangeSteps (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/nodes/HTMLStyleElement-impl.js:36:12)
    at HTMLStyleElementImpl._insert (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/nodes/Node-impl.js:824:14)
    at HTMLStyleElementImpl._replaceAll (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/nodes/Node-impl.js:1049:12)
    at HTMLStyleElementImpl.set textContent [as textContent] (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/nodes/Node-impl.js:574:14)
    at HTMLStyleElement.set textContent [as textContent] (/home/yegor/code/draft/node_modules/.pnpm/jsdom@22.1.0/node_modules/jsdom/lib/jsdom/living/generated/Node.js:678:44)
    at load (/home/yegor/code/draft/node_modules/.pnpm/primereact@10.0.9_@types+react@18.2.36_react-dom@18.2.0_react@18.2.0/node_modules/primereact/hooks/hooks.cjs.js:1009:34)
    at /home/yegor/code/draft/node_modules/.pnpm/primereact@10.0.9_@types+react@18.2.36_react-dom@18.2.0_react@18.2.0/node_modules/primereact/componentbase/componentbase.cjs.js:330:5
    at commitHookEffectListMount (/home/yegor/code/draft/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js:23150:26) 
@layer primereact {
    .p-component, .p-component * {
        box-sizing: border-box;
    }

    .p-hidden {
        display: none;
    }

    .p-hidden-space {
        visibility: hidden;
    }

    .p-reset {
        margin: 0;
        padding: 0;
        border: 0;
        outline: 0;
        text-decoration: none;
        font-size: 100%;
        list-style: none;
    }
    ... etc`

**Comment by FlipWarthog** [comment]
See this issue for more info: https://github.com/primefaces/primereact/issues/5156

tl;dr is that is a JSDOM issue and there are steps that can be taken to suppress those logs.

**Reply by iamyegor** [reply, marked as answer]
@FlipWarthog 
Thanks a lot, [this solution](https://github.com/primefaces/primevue/issues/4512#issuecomment-1749633215) helped me to resolve the issue.
